org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/xsd/participants/XSDErrorCode.java

```diff
@@ -128,7 +129,17 @@ public static Range toLSPRange(XMLLocator location, XSDErrorCode code, Object[]
 		}
 		case s4s_elt_character:
 			return XMLPositionUtility.selectContent(offset, document);
-		case src_ct_1: 
+		case sch_props_correct_2: {
+			String attrName = (String) arguments[0];
+			if (attrName.charAt(0) == ',') {
+				attrName = attrName.substring(attrName.indexOf(",") + 1);
+			} else {
+				attrName = attrName.replace(',', ':');

```

Comment:
I have never seen case with prefix in xs:element/@name (but perhaps I'm wrong). So please remove this case. If we find the case with prefix in the future, we will fix it and add a dedicated tests.

Reply:
Please ignore my previous comment, I found a case with : in element name. We should add a test with that.